components/ws-manager-bridge/src/workspace-instance-controller.ts

```diff
@@ -0,0 +1,308 @@
+/**
```

Comment:
I'm assuming this is just a lift from the original file, are there any significant changes to this file?